docs/notebooks/getting_started.ipynb

```diff
@@ -0,0 +1,1918 @@
+{
```

Comment:
<p>Maybe add here an example on how to change obs/var names? E.g. changing var_names to gene IDs </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>hm, good idea, I also thought about that because we discussed it. </p><p>Do you know of something simple that one can use to go from gene names to e.g. ensembl IDs automatically?</p>

Reply:
<p>quickly talked to @gtca, we agreed that doing the conversion with an external package is a bit of an overkill here - usually 10X data should ship the IDs with the data, so I would just add it to the list of things we want to change/improve for the example data we use here (have raw and transformed counts in there from the beginning, get rid of the weird "log-transformed raw counts" that are currently in the <code>adata</code> returned  by <code>pbmc3k_processed()</code> , remove the <code>.raw</code> object, ...etc)  </p>

Reply:
<p>https://scanpy.readthedocs.io/en/stable/generated/scanpy.queries.biomart_annotations.html</p><p>should be able to do this? </p>

Reply:
Facilitating mapping gene identifiers will also be a topic of the next hackathon. So until we have a good solution there I agree that

>  usually 10X data should ship the IDs with the data, so I would just add it to the list of things we want to change/improve for the example data we use here

is a good solution. Would you mind creating an issue?

Reply:
<p>I tried <code>scanpy.queries.biomart_annotations</code> to translate the gene names to ENS-ids, but could not match ~2000 genes on first try, so I decided this is too complicated for now and I rather mention it in the improved example data issue that I will create</p>